app/views/options-on-interstitial/check-state-pension-1.html

```diff
@@ -0,0 +1,118 @@
+{% extends "layout.html" %}
+
+{% block page_title %}
+  Start page example
```

Comment:
Users of assistive tech use titles to know what page they have landed on, we probably need to think what this should be.

Reply:
I've changed this to "Check your state pension"